happypose_ros/happypose_ros/camera_wrapper.py

```diff
@@ -146,17 +171,114 @@ def _on_image_data_cb(
         :param info: Camera info message
         :type info: sensor_msgs.msg.CameraInfo
         """
+        self._on_image_with_depth_data_cb(image, info, None, None)
+
+    def _on_image_with_depth_data_cb(
+        self,
+        color_image: Union[Image, CompressedImage],
+        color_camera_info: CameraInfo,
+        depth_image: Union[None, Image, CompressedImage] = None,
+        depth_camera_info: Union[None, CameraInfo] = None,
+    ) -> None:
+        """Called on every time synchronized image and camera info are received.
+        Saves the image and checks if intrinsics are correct. If all checks pass
+        calls :func:`_image_sync_hook`.
+
+        :param color_image: Image received from the color camera sensor
+        :type color_image: Union[sensor_msgs.msg.Image, sensor_msgs.msg.CompressedImage]
+        :param color_info: Camera info message for the color camera sensor.
+        :type color_info: sensor_msgs.msg.CameraInfo
+        :param color_image: Image received from the depth camera sensor. None if not used.
+        :type color_image: Union[None, sensor_msgs.msg.Image, sensor_msgs.msg.CompressedImage]
+        :param color_info: Camera info message for the depth camera sensor. None if not used.
+        :type color_info: Union[None, sensor_msgs.msg.CameraInfo]
```

Comment:
```suggestion
        :param depth_image: Image received from the depth camera sensor. None if not used.
        :type depth_image: Union[None, sensor_msgs.msg.Image, sensor_msgs.msg.CompressedImage]
        :param depth_info: Camera info message for the depth camera sensor. None if not used.
        :type depth_info: Union[None, sensor_msgs.msg.CameraInfo]
```

Reply:
You wrote `depth_info` and not `depth_camera_info` so I couldn't accept the suggestion. Fixed in https://github.com/agimus-project/happypose_ros/pull/23/commits/6aa0d763fdd2ca4a3d492449120a3fe4038ef8ad tho